{"uid":"4fd8f53be39998a4","name":"test_auth_user_with_valid_data","fullName":"tests.authorization.test_auth.TestAuthUser#test_auth_user_with_valid_data","historyId":"30ac640d30c0d0a91a925465a1447fed","time":{"start":1755005678519,"stop":1755005678524,"duration":5},"description":"\n        Steps.\n        1. Register new user\n        2. Try to access to store with valid data\n        3. Check that status code is 200\n        4. Check response\n        ","descriptionHtml":"<pre><code>    Steps.\n    1. Register new user\n    2. Try to access to store with valid data\n    3. Check that status code is 200\n    4. Check response\n</code></pre>\n","status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"_session_faker","time":{"start":1755005678507,"stop":1755005678508,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"attachmentsCount":0,"shouldDisplayMessage":false,"attachmentStep":false,"stepsCount":0,"hasContent":false},{"name":"register_user","time":{"start":1755005678508,"stop":1755005678519,"duration":11},"status":"passed","steps":[],"attachments":[],"parameters":[],"attachmentsCount":0,"shouldDisplayMessage":false,"attachmentStep":false,"stepsCount":0,"hasContent":false},{"name":"app","time":{"start":1755005678508,"stop":1755005678508,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"attachmentsCount":0,"shouldDisplayMessage":false,"attachmentStep":false,"stepsCount":0,"hasContent":false}],"testStage":{"description":"\n        Steps.\n        1. Register new user\n        2. Try to access to store with valid data\n        3. Check that status code is 200\n        4. Check response\n        ","status":"passed","steps":[],"attachments":[{"uid":"c14bb377e2c7a95b","name":"log","source":"c14bb377e2c7a95b.txt","type":"text/plain","size":869}],"parameters":[],"attachmentsCount":1,"shouldDisplayMessage":false,"attachmentStep":false,"stepsCount":0,"hasContent":true},"afterStages":[],"labels":[{"name":"tag","value":"positive"},{"name":"parentSuite","value":"tests.authorization"},{"name":"suite","value":"test_auth"},{"name":"subSuite","value":"TestAuthUser"},{"name":"host","value":"ixx"},{"name":"thread","value":"27672-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.authorization.test_auth"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"49ffd8689ec803b0","status":"broken","statusDetails":"requests.exceptions.ConnectionError: HTTPConnectionPool(host='127.0.0.1', port=8080): Max retries exceeded with url: /register (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001FDAC05C8C0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))","time":{"start":1755005536625,"stop":1755005536625,"duration":0}}],"categories":[],"tags":["positive"]},"source":"4fd8f53be39998a4.json","parameterValues":[]}